### PR TITLE
Unused math code is a square root function + Documented FloorLog_Unk/ConvertCollisionArray from Sonic 2 disassembly

### PR DESCRIPTION
**Pull request by DevsArchive**

### CODE REVIEW

File: _incObj/sub CalcSine.asm [hunk 1]
```diff
@@ -24,39 +24,4 @@ CalcSine:
...
-	loc_2C9A:
-		addq.w	#1,d0
-		dbf	d2,loc_2C80
-		lsr.w	#1,d0
-		movem.l	(sp)+,d1-d2
-		rts	
-		else
-		endc
\ No newline at end of file
+Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360� sine wave
```
Comment (flamewing) [inline _incObj/sub CalcSine.asm:27]: Something weird happened in this line. I am guessing your editor saved the file with wrong encoding?

File: _incObj/sub CalcSqrt.asm [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+; ===========================================================================
+
+	loc_2C9A:
+		addq.w	#1,d0
+		dbf	d2,loc_2C80
+		lsr.w	#1,d0
+		movem.l	(sp)+,d1-d2
+		rts	
+		else
+		endc
```
Comment (flamewing) [inline _incObj/sub CalcSqrt.asm:44]: New line at the end of file, maybe?

File: _incObj/sub CalcSine.asm [hunk 3]
```diff
@@ -24,39 +24,6 @@ CalcSine:
 
 ; ===========================================================================
 
-Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360� sine wave
+Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360º sine wave
```
Comment (Brainulator9) [inline _incObj/sub CalcSine.asm:27]: The line uses º, which is apparently [an ordinal indicator](https://en.wiktionary.org/wiki/%C2%BA); perhaps this should be °, [the degree sign](https://en.wiktionary.org/wiki/%C2%B0)?

File: _incObj/sub CalcSqrt.asm [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		if Revision=0
+CalcSqrt:
+		movem.l	d1-d2,-(sp)
+		move.w	d0,d1
+		swap	d1
+		moveq	#0,d0
+		move.w	d0,d1
+		moveq	#7,d2
+
+	loc_2C80:
```
Comment (Brainulator9) [inline _incObj/sub CalcSqrt.asm:23]: Perhaps loc_2C80 could be renamed to "@loop" and loc_2C9A to "@jump"?

Comment (Brainulator9) [inline _incObj/sub CalcSqrt.asm:23]: Perhaps rename loc_2C80 to loop and loc_2C9A to jump, both prefixed with at symbols?

**Pull request merged**